scripts/buildkite/main/tag-release-candidate.sh

```diff
@@ -0,0 +1,12 @@
+#! /usr/bin/env bash
+
+set -euox pipefail
+
+TAG="release-candidate-tag"
+date=$(git show -s --format=%ci | awk '{print $1}')
+DTAG="$TAG/$date"
+
+git remote set-url origin \
+  "https://$GITHUB_TOKEN@github.com/cardano-foundation/cardano-wallet.git"
+git tag "$DTAG" "$BUILDKITE_COMMIT"
+git push origin "$DTAG"
```

Comment:
```suggestion
git push origin "$DTAG"

```

(newline at end of file)